<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppQuestBDD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mesressources.qrc">
    <normaloff>:/img /Ressources/img/fleur.png</normaloff>:/img /Ressources/img/fleur.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>DTB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBDD"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Cantarell</family>
             </font>
            </property>
            <property name="text">
             <string>Table</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxTables">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>10000</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Field</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetChamps">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Enter query</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditRequete"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButtonVerif">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonHist">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Query Saving</string>
            </property>
            <property name="icon">
             <iconset resource="mesressources.qrc">
              <normaloff>:/img /Ressources/img/list-add.png</normaloff>:/img /Ressources/img/list-add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>History query</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetSelectReq"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSave">
            <property name="text">
             <string>&amp;Save History</string>
            </property>
            <property name="icon">
             <iconset resource="mesressources.qrc">
              <normaloff>:/img /Ressources/img/filesave.png</normaloff>:/img /Ressources/img/filesave.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>641</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Result</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetResultat"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonExport">
            <property name="text">
             <string>&amp;Export</string>
            </property>
            <property name="icon">
             <iconset resource="mesressources.qrc">
              <normaloff>:/img /Ressources/img/filesave.png</normaloff>:/img /Ressources/img/filesave.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_Load"/>
    <addaction name="action_Quit"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="action_Load">
   <property name="icon">
    <iconset resource="mesressources.qrc">
     <normaloff>:/img /Ressources/img/document-open.png</normaloff>:/img /Ressources/img/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="mesressources.qrc">
     <normaloff>:/img /Ressources/img/gtk-quit.png</normaloff>:/img /Ressources/img/gtk-quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="mesressources.qrc">
     <normaloff>:/img /Ressources/img/list-add.png</normaloff>:/img /Ressources/img/list-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mesressources.qrc"/>
 </resources>
 <connections/>
</ui>
